nncf/onnx/graph/nncf_graph_builder.py

```diff
@@ -186,16 +185,21 @@ def create_nncf_graph(onnx_model: ModelProto) -> NNCFGraph:
         onnx_graph = ONNXGraph(onnx_model)
         for node in onnx_graph.get_all_nodes():
             metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node.op_type)
-            if metatype is not UnknownMetatype:
-                if metatype.get_subtypes():
-                    subtype = metatype.determine_subtype(onnx_model, node)
-                    if subtype is not None:
-                        metatype = subtype
-            layer_attributes = ONNXExtendedLayerAttributes(node.input, node.output)
-            is_shared, layer_name = None, None
+            if metatype.get_subtypes():
+                subtype = metatype.determine_subtype(onnx_model, node)
+                if subtype is not None:
+                    metatype = subtype
+
             if metatype in WEIGHT_LAYER_METATYPES:
                 is_shared = onnx_graph.is_node_shared(node)
                 layer_name = onnx_graph.get_node_layer_name(node)
+                weight_shape = onnx_graph.get_weight_tensor(node)[1].shape
+                layer_attributes = ONNXWeightedNodesLayerAttributes(node.input, node.output,
```

Comment:
Why not to use `ONNXExtendedLayerAttributes`?

Reply:
Done, please check